### DISCUSSION TITLE
Will PyG support hyper-graph?

### DISCUSSION THREAD

**Discussion by xiachenrui** [pyg-team/pytorch_geometric#6114] [Q&A] [answered]
Hi all,
   PyG is really good and useful now. But it seems that it has not supported hyper-graph data structure yet. I'd like to know if it will be avaliable in the feature or it has been supported in other ways now ?

Best!

**Comment by rusty1s** [comment, marked as answer]
Yeah, that is mostly correct. We only provide a single operator for this (`HyperGraphConv`), and no example nor tutorial on how to create and work with hypergraphs. Happy to change this if you have something specific in mind :)

**Comment by xiachenrui** [comment]
Thank you! I find that the package [DHG(DeepHypergraph)](https://deephypergraph.readthedocs.io/en/latest/index.html) support [hypergraph](https://deephypergraph.readthedocs.io/en/latest/api/dhg.html#dhg.Hypergraph) now. If it is possible to incorporate it into the PyG ecosystem in the feature ?

**Reply by rusty1s** [reply]
You mean as a conversion utility function? Sure that's possible. Do you wanna help with that?
